### DISCUSSION TITLE
Video durations on Subscriptions view have gone

### DISCUSSION THREAD

**Discussion by garnercx** [FreeTubeApp/FreeTube#2792] [Q&A]
# Missing Video Durations

## Problem Description
I really hope this isn't a stupid user error, but the durations of all the videos have disappeared for me on the Subscriptions tab. They appear in every individual subscribed channel, and other tabs such as History and Trending, but not Subscriptions. Anyone know what I might be doing wrong here? Happy to move to a bug in Issues if that turns out to be what it is.

## Screenshots

Attached files:
1. The Subscriptions view with the video durations missing <img width="1361" alt="Freetube Screenshot - Subscriptions Tab Showing Missing Video Durations" src="https://user-images.githubusercontent.com/111322/198995593-b691e0ec-112e-4bf3-a0f8-d525441f0f78.png">
2. A channel with the video durations in place. <img width="1364" alt="Freetube Screenshot - Channel Subscription Showing Video Durations" src="https://user-images.githubusercontent.com/111322/198995634-3278d9a0-20a3-4139-bffd-1b4039456923.png">

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi there, we just release v0.18.0. Are u still facing this issue in v0.18.0?

**Comment by garnercx** [comment]
Yep - all fixed in v0.18.0. Thanks for the new release.
